src/System.Security.Cryptography.Algorithms/tests/RandomNumberGeneratorTests.cs

```diff
@@ -151,7 +151,7 @@ private static void DifferentSequential(int arraySize)
             // = 1/256 * 1/256 * ... * 1/256
             // = 1/(256^10)
             // = 1/1,208,925,819,614,629,174,706,176
-            Assert.NotEqual(first, second);
+            Assert.NotEqual<byte>(first, second);
```

Comment:
Does this have a measurable change for you?  I just verified that Equal, at least, seems to do what I want:

```C#
byte[] data = ByteUtils.RepeatByte(0x71, 11);
byte[] data2 = (byte[])data.Clone();
data2[0] = 3;

Assert.Equal(data, data2);
```


```
 System.Security.Cryptography.EcDsaOpenSsl.Tests.EcDsaOpenSslTests.VerifyDuplicateKey_ValidHandle [FAIL]
      Assert.Equal() Failure
      Expected: Byte[] [113, 113, 113, 113, 113, ...]
      Actual:   Byte[] [3, 113, 113, 113, 113, ...]
      Stack Trace:
            at System.Security.Cryptography.EcDsaOpenSsl.Tests.EcDsaOpenSslTests.VerifyDuplicateKey_ValidHandle()
```